sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluator.java

```diff
@@ -87,6 +90,7 @@ public OfferEvaluator(
       FrameworkStore frameworkStore,
       StateStore stateStore,
       Optional<OfferOutcomeTracker> offerOutcomeTracker,
+      Optional<OfferOutcomeTrackerV2> offerOutcomeTrackerV2,
```

Comment:
(not sure if this has already been discussed) Considering that the current debug endpoint is for debugging only, i think we can just remove this completely in favor of the new endpoint(s). It's not like the user would be losing some information as the enriched information is present at the new endpoint anyway. wdyt ?

Reply:
ya I agree, however according to the current plan we will completely cut the v1 endpoint once the UI is using v2, and we have fine tuned the v2 endpoint after some trial. This isnt an issue for now, im for keeping it and dropping v1 in a separate PR.